_posts/cloud/2021-01-29-OpenStack.md

```diff
@@ -10,8 +10,61 @@ project:
   title: "OpenStack"
   type: "Python"
   ci_url: "https://zuul.opendev.org/t/openstack/status"
-  url: "https://zuul.opendev.org/t/openstack/status"
+  url: "OpenStack"
```

Comment:
is this intentional ?
It is not really a URL anymore.

Reply:
Maybe use https://www.openstack.org ?

Reply:
Yes, OpenStack will join others and become the url like this:
https://kunpengcompute.github.io/arm-landscape/OpenStack/